### PR TITLE
Add Load Paths docs to the active_admin.rb template

### PR DESCRIPTION
**Pull request by gabo-cs**
It would be nice to have this piece of doc in the `active_admin.rb` template, I know it's already in the [doc site](https://activeadmin.info/1-general-configuration.html#load-paths), but sometimes we take a look directly at the initializer comments to look for useful examples.

### CODE REVIEW

File: lib/generators/active_admin/install/templates/active_admin.rb.erb [hunk 1]
```diff
@@ -18,6 +18,21 @@ ActiveAdmin.setup do |config|
   #
   # config.site_title_image = "logo.png"
 
+  # == Load Paths
+  #
+  # By default Active Admin files go inside app/admin/.
+  # You can change this directory.
+  # eg:
+  # config.load_paths = [File.join(Rails.root, 'app', 'ui')]
```
Comment (deivid-rodriguez) [inline lib/generators/active_admin/install/templates/active_admin.rb.erb:26]:
```suggestion
  #
  # eg:
  #   config.load_paths = [File.join(Rails.root, 'app', 'ui')]
```

To match the formatting of "Default namespace" docs below.

File: lib/generators/active_admin/install/templates/active_admin.rb.erb [hunk 2]
```diff
@@ -18,6 +18,21 @@ ActiveAdmin.setup do |config|
...
+  #
+  # Or, you can also load more directories.
+  # Useful when setting namespaces with users that are not your main AdminUser entitiy.
+  # eg:
+  # config.load_paths = [
+  #   File.join(Rails.root, 'app', 'admin'),
+  #   File.join(Rails.root, 'app', 'cashier')
+  # ]
```
Comment (deivid-rodriguez) [inline lib/generators/active_admin/install/templates/active_admin.rb.erb:34]:
```suggestion
  #
  # eg:
  #   config.load_paths = [
  #     File.join(Rails.root, 'app', 'admin'),
  #     File.join(Rails.root, 'app', 'cashier')
  #   ]
```

Ditto.

File: lib/generators/active_admin/install/templates/active_admin.rb.erb [hunk 3]
```diff
@@ -18,6 +18,21 @@ ActiveAdmin.setup do |config|
...
 
+  # == Load Paths
+  #
+  # By default Active Admin files go inside app/admin/.
+  # You can change this directory.
+  # eg:
+  # config.load_paths = [File.join(Rails.root, 'app', 'ui')]
+  #
+  # Or, you can also load more directories.
+  # Useful when setting namespaces with users that are not your main AdminUser entitiy.
```
Comment (deivid-rodriguez) [inline lib/generators/active_admin/install/templates/active_admin.rb.erb:29]:
```suggestion
  # Useful when setting namespaces with users that are not your main AdminUser entity.
```

**Review by deivid-rodriguez — COMMENTED** [review]
Minor edits, looks good otherwise.

**Comment by gabo-cs** [comment]
@deivid-rodriguez Thanks! Suggestions commited!

**Review by deivid-rodriguez — APPROVED** [review]

**Pull request merged**